components/ProjectDefault.tsx

```diff
@@ -27,47 +27,58 @@ export default function ProjectDefault({
     const opacityProgress = useTransform(scrollYProgress, [0, 1], [0.6, 1]);
 
     return (
-        // project card is a section
-        <motion.div
-            ref={ref}
-            style={{
-                scale: scaleProgress,
-                opacity: opacityProgress,
-            }}
-            className='mb-3 group sm:mb-8 max-w-[60%] last:mb-0'
-        >
-            <section className='hover:bg-gray-200 group pb-3 bg-gray-100 group-even:pl-8 rounded-lg border border-black/5 overflow-hidden sm:pr-8 relative sm:h-[20rem] sm:mb-8 mb-3 last:mb-0'>
-                <div className='px-5 py-4 mt-4 pb-7 sm:pl-10 group-even:ml-[21rem] even:pl-8 sm:pr-2 sm-pt-10 sm:max-w-[50%] flex flex-col h-full'>
-                    <h3 className='text-2xl font-semibold'>{title}</h3>
-                    <p className='mt-2 leading-relaxed text-gray-700'>
-                        {description} <br />
-                        {/* <Link
-                            className=' text-gray-400 pb-3 font-semibold hover:underline cursor-pointer'
-                            href={blogLink}
-                        >
-                            Read more
-                        </Link> */}
-                    </p>
-                    <ul className='flex flex-wrap gap-2 mt-4 sm:mt-auto'>
-                        {tags.map((tag, index) => (
-                            <li
-                                key={index}
-                                className='bg-black/0.7 px-3 py-1 text-[0.7rem] uppercase tracking-wider bg-gray-700 text-white rounded-full'
-                            >
-                                {tag}
-                            </li>
-                        ))}
-                    </ul>
+        <>
+            <motion.div
+                ref={ref}
+                style={{
+                    scale: scaleProgress,
+                    opacity: opacityProgress,
+                }}
+                className='mb-3 group sm:mb-8 max-w-full sm:max-w-[60%] last:mb-0'
+            >
+                <Link
+                    className='cursor-pointer'
+                    href={blogLink as LinkProps['href']}
```

Comment:
Unnecessary casting of blogLink as LinkProps['href']

href will accept a string anyways

Reply:
Odd, You sure it's type string? I believe you

Reply:
Odd, You sure it's type string? I believe you